dao/src/main/scala/za/co/absa/enceladus/dao/rest/CrossHostApiCaller.scala

```diff
@@ -60,6 +69,8 @@ protected class CrossHostApiCaller private(apiBaseUrls: Vector[String], maxTryCo
 
 
   def call[T](fn: String => T): T = {
+    var canBeRecovered = false
```

Comment:
Let's try to avoid using this vars

Reply:
I know, I hate it :( it's because I couldn't find an easy solution for handling two errors that are recoverable by default: `ResourceAccessException` and `RestClientException` so now the logic is mixed. Probably I/we had to think about different design of code related to this.